src/test/java/pl/byczazagroda/trackexpensesappbackend/service/WalletServiceImplTest.java

```diff
@@ -28,6 +36,8 @@
 class WalletServiceImplTest {
 
     private static final String NAME_OF_WALLET = "nameOfWallet";
+    private static final String NAME_OF_WALLET_1 = "nameOfWallet1";
+    private static final String NAME_OF_WALLET_2 = "nameOfWallet2";
```

Comment:
It seems to me that in the previous Test you had individual constants separated by a blank line. So let's keep one common convention.

Reply:
I suggested the naming convention for the constants created in the previous PR